src/inference/include/ie/vpu/myriad_config.hpp

```diff
@@ -24,6 +24,8 @@ namespace InferenceEngine {
  */
 DECLARE_VPU_CONFIG(MYRIAD_ENABLE_FORCE_RESET);
 
+DECLARE_VPU_CONFIG(MYRIAD_DISABLE_MX_BOOT);
```

Comment:
please, provide doxygen documentation for this option

Reply:
Done, thank you.